src/Symfony/Component/ExpressionLanguage/Tests/ExpressionLanguageTest.php

```diff
@@ -245,6 +245,13 @@ public function testCallBadCallable()
         $el->evaluate('foo.myfunction()', ['foo' => new \stdClass()]);
     }
 
+    public function testCallBadCallableWithNullSafe()
+    {
+        $el = new ExpressionLanguage();
+        $result = $el->evaluate('foo?.myfunction()', ['foo' => new \stdClass()]);
```

Comment:
This test is not correct: `foo?.myfunction()` should mean that if `foo` is null or undefined, then `null` should be returned.
this is unrelated to the method existing or not